tests/CMakeLists.txt

```diff
@@ -82,12 +82,13 @@ set(PYBIND11_TEST_FILES
     test_pickling.cpp
     test_pytypes.cpp
     test_sequences_and_iterators.cpp
-    test_smart_ptr.cpp
+    #AV  this fails with PGI test_smart_ptr.cpp
     test_stl.cpp
     test_stl_binders.cpp
     test_tagbased_polymorphic.cpp
     test_union.cpp
-    test_virtual_functions.cpp)
+    #AV  this fails with PGI test_virtual_functions.cpp
```

Comment:
These should be removed if compiler is PGI. I can add that if you want me to.

Reply:
Would be great!
I mean I haven't found an easy way ot exclude the execution of the python tests.